CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Keycloak Community
+
+Keycloak Operator is an Open Source solution to easily deploy and manage Keycloak server instances.
+
+## Building and working with the codebase
+
+Details for building from source and working with the codebase are provided in the [building and working with the code base](docs/building.md) guide.
+
+## Contributing to the Keycloak Operator
+
+The Keycloak Operator is an Open Source community-driven project and we welcome contributions as well as feedback from the community.
+
+We do have a few guidelines in place to help you be successful with your contribution.
+
+Here's a quick checklist for a good PR, more details below:
+
+1. [Keycloak Dev Mailing List](https://groups.google.com/forum/#!forum/keycloak-dev)
+2. A JIRA associated with the PR
+3. One feature/change per PR
+4. One commit per PR
+5. PR rebased on master (`git rebase`, not `git pull`) 
+5. Commit message is prefixed by JIRA number
+6. No changes to code not directly related to your PR
+7. Includes test
+8. Includes documentation
+
+Once you have submitted your PR please monitor it for comments/feedback. We reserve the right to close inactive PRs if
+you do not respond within 2 weeks (bear in mind you can always open a new PR if it is closed due to inactivity).
+
+Also, please remember that we do receive a fairly large amount of PRs and also have code to write ourselves, so we may
+not be able to respond to your PR immediately. The best place to ping us is on the thread you started on the dev mailing list.
+
+### Finding something to work on
+
+If you would like to contribute to Keycloak, but are not sure exactly what to work on, you can find a number of open
+issues that are awaiting contributions in the 
+[Keycloak JIRA](https://issues.jboss.org/projects/KEYCLOAK/versions/12340167).
```

Comment:
I believe this link could be updated to get all issues with `Containers - Operator` component.

Reply:
@slaskawi I tried to keep in sync with the same information from other repositories. But if you and @stianst think we should change, np at all.